content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesHelperAction.java

```diff
@@ -654,9 +654,6 @@ protected String buildUploadFilesContext(VelocityPortlet portlet, Context contex
 		String upload_limit = rb.getFormattedMessage("upload.limit", new String[]{ max_file_size_mb });
 		context.put("upload_limit", upload_limit);
 
-		String uploadWarning = rb.getFormattedMessage("label.overwrite.warning");
```

Comment:
Side node (doesn't need changing) but it's always better to just put the resource loader into the context and then to get the strings directly from  it in the template. Leads to less code.